catalogue/webapp/pages/works.js

```diff
@@ -272,17 +291,44 @@ const Works = ({
                           });
                         }}
                       >
-                        <WorkCard
-                          work={result}
-                          params={{
-                            ...searchParams,
-                            id: result.id,
-                          }}
-                        />
+                        {searchParams.imageSearch ? (
+                          <>
+                            <ImageCard
+                              id={result.id}
+                              image={{
+                                contentUrl: result.thumbnail
+                                  ? result.thumbnail.url
+                                  : 'https://via.placeholder.com/1600x900?text=%20',
+                                width: 300,
+                                height: 300,
+                                alt: result.title,
+                                tasl: null,
+                              }}
+                              onClick={() => setExpandedImageId(result.id)}
+                            />
+                            {expandedImageId === result.id && (
+                              <ExpandedImage
+                                index={i}
+                                title={result.title}
+                                id={result.id}
+                                searchParams={searchParams}
+                              />
+                            )}
+                          </>
+                        ) : (
+                          <WorkCard
+                            work={result}
+                            params={{
+                              ...searchParams,
+                              id: result.id,
+                            }}
+                          />
+                        )}
```

Comment:
Bikeshed - wonder if there's a point in making a `<WorkResults />` and `<ImageResults>` component.

Reply:
Considered it but since they're v small I didn't bother - back in the bad old React classes days I'd have done a `this.renderResults()` method but that feels wrong now...